Taarafo.Core/Services/Foundations/Profiles/ProfileService.cs

```diff
@@ -29,11 +29,18 @@ public partial class ProfileService : IProfileService
         }
 
         public ValueTask<Profile> AddProfileAsync(Profile profile) =>
-            TryCatch(async () =>
+        TryCatch(async () =>
         {
             ValidateProfileOnAdd(profile);
 
             return await this.storageBroker.InsertProfileAsync(profile);
         });
+
+        public IQueryable<Profile> RetrieveAllProfiles() =>
+        TryCatch(() =>
+        {
+            return this.storageBroker.SelectAllProfiles();
+
```

Comment:
Take this line out. And one-line this whole statement please

Reply:
done